specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2021-10-01/imagebuilder.json

```diff
@@ -0,0 +1,1496 @@
+      ]
+    },
+    "ImageTemplateIdentity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the image template. The type 'None' will remove any identities from the image template.",
+          "enum": [
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "properties": {
+              "principalId": {
+                "readOnly": true,
+                "type": "string",
+                "description": "The principal id of user assigned identity."
+              },
+              "clientId": {
+                "readOnly": true,
+                "type": "string",
+                "description": "The client id of user assigned identity."
+              }
+            }
+          },
+          "description": "The list of user identities associated with the image template. The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'."
+        }
+      },
+      "description": "Identity for the image template."
+    },
+    "RunOutputProperties": {
+      "type": "object",
+      "description": "Describes the properties of a run output",
+      "properties": {
+        "artifactId": {
+          "type": "string",
+          "description": "The resource id of the artifact."
+        },
+        "artifactUri": {
+          "type": "string",
+          "description": "The location URI of the artifact."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning state of the resource",
+          "readOnly": true
+        }
+      }
+    },
+    "ImageTemplate": {
+      "type": "object",
+      "description": "Image template is an ARM resource managed by Microsoft.VirtualMachineImages provider",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ImageTemplateProperties",
+          "description": "The properties of the image template"
+        },
+        "identity": {
+          "$ref": "#/definitions/ImageTemplateIdentity",
+          "description": "The identity of the image template, if configured."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData",
+          "description": "Metadata pertaining to creation and last modification of the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
```

Comment:
As this is new api version, I would suggest to reference `Resource` model from common-types.

Reply:
@raych1  The "Resource" model  from common-types does not contain 'location' and 'tags', it will fail model validation as well. Previously we have duplicated model definition reference and it worked well, I do not know what happened to the checks but it stopped working for us and begin to give these errors. Please advise on this, currently we heavily depend on model definition reference, including the usage of 'allOf' label.